salsah/src/test/scala/org.knora.salsah/browser/SalsahPage.scala

```diff
@@ -88,7 +96,7 @@ class SalsahPage {
       * @param password user's password
       * @param fullName user's full name
       */
-    def doLogin(email: String, password: String, fullName: String): Unit = {
+    def doLogin(email: String, password: String, fullName: String, _page: SalsahPage): Unit = {
```

Comment:
Why does `page.doLogin` need to get passed a reference to `page`? Isn't that just `this`?